src/components/activity/ActivityPlanning.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useMemo } from 'react';
+import { View } from 'react-native';
+
+import { CalendarEntry } from 'storage/types';
+import Storage from 'storage';
+import { CalendarList } from '../calendar/CalendarList';
+import { ISODate, ISOTime } from 'utils';
+import { ConfrimPortal } from './ConfirmPortal';
```

Comment:
The component should be renamed to ConfirmPortal

Reply:
It might just be me, but is'nt it already named ConfirmPortal?

Reply:
Right not it is spelled Confrim when it should be spelled Confirm